src/libs/getOperatingSystem/index.js

```diff
@@ -0,0 +1,26 @@
+import CONST from '../../CONST';
+
+/**
+ * Reads the current operating system when running on Web/Mobile-Web/Desktop
+ * @return {String | null}
+ */
+export default () => {
+    const {userAgent, platform} = window.navigator;
+    const macosPlatforms = ['Macintosh', 'MacIntel', 'MacPPC', 'Mac68K'];
+    const windowsPlatforms = ['Win32', 'Win64', 'Windows', 'WinCE'];
+    const iosPlatforms = ['iPhone', 'iPad', 'iPod'];
```

Comment:
NAB, if we only needed to know that we're on windows we could have created an `isUsingWindows` helper instead.